### ISSUE TITLE
flex class is weird.

### ISSUE THREAD

**Issue by m0ksem** [epicmaxco/vuestic-ui#1927]
So there's this style that's being applied by vuestic:

```css
.flex>* {
    flex: 1 1 auto;
}
```
Would be nice to be able to either turn it off, or have this being applied in a different way, I think it just messes up with other UIs that may use flexbox.
I tried reverting it like so in Nuxt 3's app.vue
```html
<style>
.flex > * {
    flex: revert;
}
</style>
```
But it doesn't seem to really help and some elements in my site are still messed up as I use flex a lot.

I don't know if there are more examples of such styles in this framework, but it's possible it's better to have these not applied globally, but by classes provided by vuestic's components.

_Originally posted by @Luffyyy in https://github.com/epicmaxco/vuestic-ui/discussions/1917_

**Comment by wa1id** [comment]
This messes with Tailwind for me

**Comment by m0ksem** [comment]
We will fix it in 1.5.0 with css code split rework.
https://github.com/epicmaxco/vuestic-ui/pull/1801

**Comment by m0ksem** [comment]
It is possible to remove grid.css after 1.4.7.
https://vuestic.dev/en/getting-started/tree-shaking
